lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h

```diff
@@ -68,7 +70,7 @@ XX(identifier, string, none, identifier, __VA_ARGS__)\
 XX(path, string, none, path, __VA_ARGS__)
 
 #define TNL_DISABLE_IDENTITY(XX, ...) \
-XX(path, string, none, path, __VA_ARGS__)
+XX(identifier, string, none, id, __VA_ARGS__)
```

Comment:
`identifier` isn't quite right here, is it? Why not call it "identity"? I also think the json json and C field names should match.

Reply:
I changed the id json field to identifier

Reply:
The identity is called as Identifier in the load function, instance map and in all the IPC commands